docs/installation-configuration.md

```diff
@@ -22,7 +22,7 @@ The Pulumi `OVH` provider is available as a package in all Pulumi languages:
 The OVH provider binary is a third party binary. It can be installed using the `pulumi plugin` command.
 
 ```bash
-pulumi plugin install resource ovh <version> --server api://github.com/ovh/pulumi-ovh
+pulumi plugin install resource ovh vX.Y.Z --server https://github.com/ovh/pulumi-ovh/releases/download/vX.Y.Z/
```

Comment:
@scraly the URL here should match the one in your `resources.go` file here:

https://github.com/ovh/pulumi-ovh/blob/26f9b86620d55bcd7d9977996f524ce8593980ef/provider/resources.go#L105

We do have explicit Github Releases support documented here:

https://www.pulumi.com/docs/using-pulumi/pulumi-packages/how-to-author/#support-for-github-releases

Can you have this fixed in a new release?

Reply:
I tested it Ok I will update the README file according to this new information about the installation process